index.js

```diff
@@ -11,7 +11,8 @@ const DEVICE_TYPES = {
   'SIREN1': 'Siren',
   'SMARTPLUG': 'Smart plug',
   'SMOKE2': 'Rökdetektor',
-  'VOICEBOX1': 'Directenhet'
+  'VOICEBOX1': 'Directenhet',
+  'HUMIDITY1': 'Luftfuktighet'
```

Comment:
Yes of course. Used the Norwegian name for it. I'm not sitting with the machine with git at the moment so feel free to do the changes 👍